### ISSUE TITLE
Fatal error with 2.2.0-beta2 running under PowerShell 7.2.0-preview.9 on Raspberry Pi 2 running Raspbian Buster

### ISSUE THREAD

**Issue by sba923** [PowerShell/PSReadLine#2807]
### Environment
PSReadLine: 2.2.0-beta2
PowerShell: 7.2.0-preview.9
OS: Linux 5.10.17-v7+ #1421 SMP Thu May 27 13:59:01 BST 2021
BufferWidth: 141
BufferHeight: 31

Last 0 Keys
```

```

### Exception
```
System.TypeLoadException: Could not load type 'System.Management.Automation.Subsystem.PredictionResult' from assembly 'Microsoft.PowerShell.PSReadLine.Polyfiller, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'.
   at Microsoft.PowerShell.PSConsoleReadLine.PredictionViewBase.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.PredictionInlineView.Reset() in Microsoft.PowerShell.PSReadLine2.dll:token 0x60004ca+0x0
   at Microsoft.PowerShell.PSConsoleReadLine.Prediction.Reset() in Microsoft.PowerShell.PSReadLine2.dll:token 0x600049b+0x7
   at Microsoft.PowerShell.PSConsoleReadLine.Initialize(Runspace runspace, EngineIntrinsics engineIntrinsics) in Microsoft.PowerShell.PSReadLine2.dll:token 0x6000101+0x118
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics, CancellationToken cancellationToken) in Microsoft.PowerShell.PSReadLine2.dll:token 0x60000fa+0x60
```

## Steps to reproduce

Run PowerShell 7.2.0-preview.9 configured to use PSReadLine 2.2.0-beta2 and wait for the prompt.

Occurs even with `pwsh -noprofile`

Works fine with PowerShell 7.1.4.

Feel free to ask for more information / troubleshooting.

**Comment by daxian-dbw** [comment]
This is a known issue and has been addressed in 2.2.0-beta3 release of PSReadLine. Please upgrade to use 2.2.0-beta3 with the latest 7.2.0-preview.9 PowerShell.

**Comment by sba923** [comment]
Fix works like a charm, thanks!
